challenges/functions.js

```diff
@@ -7,26 +7,41 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+function consume (arr1, arr2, cb) {
+  return cb (arr1, arr2);
```

Comment:
Just to nitpick here (since everything looks great), it looks like you have an extra space. `function consume(arr1, arr2, cb)` and `return cb(arr1, arr2);`